tips/tests/tips_views_tests.py

```diff
@@ -101,7 +86,9 @@ def test_tips_views_templates(self):
         except TemplateDoesNotExist:
             assert False, f"Template {EDIT_TIP_PAGE_ROUTE} does not exist"
 
-    def test_delete_tip_page(self, generate_get_request, generate_tip):
+    def test_delete_tip_page(self, generate_tip):
```

Comment:
The name of this test confused me a bit
Does this test delete a tip from the page?
Or does it delete the page of the tips?
Please change it to a more meaningful name

Reply:
done